src/compat.c

```diff
@@ -454,11 +459,25 @@ struct Bdirent*	Breaddir(BDIR *dir)
 	dirr->info.size = 0;
 	dirr->info.mtime = 0;
 
+#ifdef __MINGW32__
```

Comment:
Perhaps use tabs instead of spaces here to be conform to the rest of the src file